lockbox/programs/liquidity_lockbox/src/lib.rs

```diff
@@ -401,22 +396,7 @@ pub mod liquidity_lockbox {
 
       // Close the pda_lockbox_position account and send all lamports to the receiver
       // Secure reference: https://github.com/coral-xyz/sealevel-attacks/blob/master/programs/9-closing-accounts/secure/src/lib.rs
-      let dest_starting_lamports = ctx.accounts.signer.lamports();
-
-      let account = ctx.accounts.pda_lockbox_position.to_account_info();
-      **ctx.accounts.signer.lamports.borrow_mut() = dest_starting_lamports
-        .checked_add(account.lamports())
-        .unwrap();
-      **account.lamports.borrow_mut() = 0;
-
-      let mut data = account.try_borrow_mut_data()?;
-      for byte in data.deref_mut().iter_mut() {
-        *byte = 0;
-      }
-
-      let dst: &mut [u8] = &mut data;
-      let mut cursor = Cursor::new(dst);
-      cursor.write_all(&CLOSED_ACCOUNT_DISCRIMINATOR).unwrap();
+      ctx.accounts.pda_lockbox_position.close(ctx.accounts.signer.to_account_info())?;
```

Comment:
Change manual account closing to a new `close()` function provided out of the box.